packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -321,7 +325,12 @@ class Explorer extends Foundation<
                         >
                             <Background value={neutralLayerL3} drawBackground={false}>
                                 <div className={classNames(explorer_paneTitleContainer)}>
-                                    <Heading size={HeadingSize._6}>FAST Explorer</Heading>
+                                    <Heading
+                                        size={HeadingSize._6}
+                                        title={`Version ${version}`}
```

Comment:
should we consider using `Badge`?

Reply:
We could add Badge - I meant to open as a draft initially, but here we are :)

Reply:
That would be consistent with what we did for Creator. I think that works well considering the related change to publish the latest published version.
![image](https://user-images.githubusercontent.com/47367562/72012968-cccc5f00-3211-11ea-83f3-4367a814393b.png)